src/content/developers/docs/scaling/layer-2-rollups/index.md

```diff
@@ -7,21 +7,21 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-Layer 2 is a collective term for solutions designed to help scale your application by handling transactions off the Ethereum Mainnet (layer 1), while taking advantage of the robust decentralized security model of Mainnet. Transaction speed suffers when the network is busy which can make the user experience poor for certain types of dapps. And as the network gets busier, gas prices increase as transaction senders aim to outbid each other. This can make using Ethereum very expensive.
+Layer 2 is a collective term for solutions designed to help scale your application by handling transactions off the Ethereum Mainnet (layer 1) while taking advantage of the robust decentralized security model of Mainnet. Transaction speed suffers when the network is busy, making the user experience poor for certain types of dapps. And as the network gets busier, gas prices increase as transaction senders aim to outbid each other. This can make using Ethereum very expensive.
 
 ## Prerequisites {#prerequisites}
 
-You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested, and continues to be researched and developed.
+You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested and continues to be researched and developed.
 
 ## Why is layer 2 needed? {#why-is-layer-2-needed}
 
-- Some use-cases, like blockchain games, make no sense with current transaction times
-- It can be unnecessarily expensive to use blockchain applications
+- Some use-cases, like blockchain games, make no sense with current transaction times.
+- It can be unnecessarily expensive to use blockchain applications.
 - Any updates to scalability should not be at the expense of decentralization or security – layer 2 builds on top of Ethereum.
```

Comment:
Curious, weren't we trending towards dropping the "hard stops" at the end of bullet points?

Reply:
@wackerow The rule I've been trying to follow (which is at least normal in British English) is full stops if each list item is a complete sentence and no full stops for fragments (and don't mix full sentences with fragments in the same list).

Does that seem okay to you? I'm not wildly in favour of one way or another, just consistency.

Reply:
Gotcha, yeah this works then.